### PR TITLE
Replaced GLEW by GLBrebis drop-in

### PR DESCRIPTION
**Pull request by mean-ui-thread**

### CODE REVIEW

**Comment by mean-ui-thread** [comment]
+24,759 −21. Small code change to review. ;-)

File: CMakeLists.txt [hunk 1]
```diff
@@ -339,7 +335,6 @@ if (WIN32)
         list(APPEND libs PUBLIC ${OPENGL_LIBRARIES})
 
         link_directories(./lib/win/Release/)
-        list(APPEND libs PUBLIC glew32s)
```
Comment (Daivuk) [inline CMakeLists.txt:337]: We could also delete the file in `/lib/win/Release/`

File: src/RendererGL.cpp [hunk 2]
```diff
@@ -519,7 +520,7 @@ namespace onut
                 programDirty ||
                 renderTargetDirty)
             {
-                // Because opengl sucks, we need to determine if we have to flip Y
+                // Because opengl sucks, we need to determine if we have to flip Y
```
Comment (Daivuk) [inline src/RendererGL.cpp]: I don't understand the change

File: src/onut_gl/OnutGL.c [hunk 3]
```diff
@@ -0,0 +1,9783 @@
...
+ * MATERIALS OR THE USE OR OTHER DEALINGS IN THE MATERIALS.
+ ******************************************************************************/
+
+#include <onut_gl/OnutGL.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#if defined(_WIN32) && !defined(__CYGWIN__)
```
Comment (Daivuk) [inline src/onut_gl/OnutGL.c]: Hum usually its just `WIN32` on windows not `_WIN32`. Is there something I dont know?

File: src/RendererGL.cpp (hunk 2 above)
Comment (mean-ui-thread) [reply src/RendererGL.cpp]: not sure what happened here... lol I think my IDE changed a tab to a white space.

File: src/onut_gl/OnutGL.c (hunk 3 above)
Comment (mean-ui-thread) [reply src/onut_gl/OnutGL.c]: `WIN32` is defined by the SDK or the build environment, so it does not use the implementation reserved namespace.
`_WIN32` is defined by the compiler so it uses the underscore to place it in the implementation-reserved namespace

**Comment by mean-ui-thread** [comment]
don't merge yet. I have improved GLBrebis significantly in the past few days.

File: src/onut_gl/OnutGL.c (hunk 3 above)
Comment (mean-ui-thread) [reply src/onut_gl/OnutGL.c]: So I guess there was something you didn't know :trollface:

File: src/RendererGL.cpp (hunk 2 above)
Comment (mean-ui-thread) [reply src/RendererGL.cpp]: I can try reverting that one line if you want.

File: CMakeLists.txt (hunk 1 above)
Comment (mean-ui-thread) [reply CMakeLists.txt:337]: done

**Pull request closed without merging**